integration/kubernetes/k8s-footloose.bats

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bats
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+load "${BATS_TEST_DIRNAME}/../../.ci/lib.sh"
+load "${BATS_TEST_DIRNAME}/../../lib/common.bash"
+
+setup() {
+	export KUBECONFIG="$HOME/.kube/config"
+	pod_name="footfed"
+	config_name="ssh-pub-key"
+	get_pod_config_dir
+
+	# Creates ssh-key
+	public_key_path="/tmp/mykey.pub"
```

Comment:
I know this is a test, but still - this path is predictable and therefore not "safe". Can you change this to:

```
tmpdir=$(mktemp)
public_key_path="${tmpdir}/mykey.pub"

   : 
   : 

[ -d "$tmpdir" ] && rm -rf "$tmpdir"
```